etl/steps/data/garden/owid/latest/key_indicators/table_population.py

```diff
@@ -36,7 +34,23 @@ def make_table() -> Table:
         .pipe(prepare_dataset)
     )
 
+    source = Source(
+        name="Gapminder (v6), HYDE (v3.2), UN (2019)",
+        description='Our World in Data builds and maintains a long-run dataset on population by country, region, and for the world, based on three key sources: HYDE, Gapminder, and the UN World Population Prospects. You can find more information on these sources and how our time series is constructed on this page: <a href="https://ourworldindata.org/population-sources">What sources do we rely on for population estimates?</a>',
+        url="https://www.gapminder.org/data/documentation/gd003/; https://population.un.org/wpp/Download/Standard/Population/; https://dataportaal.pbl.nl/downloads/HYDE/",
+        date_accessed="October 8, 2021",
+    )
```

Comment:
Hmm, interesting. So, in the ETL model, you can have multiple sources, with the idea that as you use different sources you concatenate them in the metadata, ending up with a list of all the sources you actually used. This is better than grapher's model. But it's also worse in that it's missing a single, overall `source_summary` that explains how the different sources were used.

I think this is fine for a backported variable, but we can probably aim for better for data that originates from the ETL.

Reply:
Oh yeah, list of sources is better for sure. (I thought I left a comment about it somewhere, but apparently not). Fixed.